src/App.tsx

```diff
@@ -8,38 +8,35 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
 import { Post } from './types/Post';
 import { Counter } from './features/counter/Counter';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { loadApiPosts, setPosts } from './features/postsSlice';
```

Comment:
looks like you did not add all changes to git
try to run `git add .`, commit and push changes again and re-request review

Reply:
You are absolutely right, i forgot about that